src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
```

Comment:
The function name 'makeCalculator' suggests that the object returned should be named 'calculator'. This is done correctly here, so good job following the naming conventions.